bin/dry-run.rb

```diff
@@ -227,6 +250,11 @@ def cached_dependency_files_read
       FileUtils.mkdir_p(files_dir) unless Dir.exist?(files_dir)
       File.write(files_path, file.content)
     end
+    # Initialize a git repo so that changed files can be diffed
+    FileUtils.cp(".gitignore", File.join(cache_dir, ".gitignore"))
+    Dir.chdir(cache_dir) do
+      system("git init . && git add . && git commit --allow-empty -m 'Init'")
```

Comment:
Interesting approach, will be super helpful!

Reply:
🤠